lang/python/python-curl/Makefile

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright (C) 2010 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-curl
+PKG_VERSION:=7.21.5
+PKG_RELEASE:=1
+PKG_MAINTAINER:=Waldemar Konik <informatyk74@interia.pl>
+PKG_LICENSE:=LGPL-2.1
+PKG_LICENSE_FILE=COPYING-LGPL
+
+PKG_SOURCE:=pycurl-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://pycurl.sourceforge.net/download/
```

Comment:
https is preferred over http
Please use https://dl.bintray.com/pycurl/pycurl/ instead.